packages/jupyter-ai/jupyter_ai/chat_handlers/base.py

```diff
@@ -79,6 +97,11 @@ class BaseChatHandler:
     specified by the config. Subclasses should define this. Should be set to
     `False` for handlers like `/help`."""
 
+    display_arguments_help: ClassVar[bool] = True
+    """Class attribute specifying whether this chat handler should
+    parse the arguments and display help when user queries with
+    `-h` or `--help`"""
+
```

Comment:
I agree with your implementation here; using an explicit class attribute to determine this behavior is a better solution than always handling help in the base class unless `self.__class__.id in ('default', 'help')`. 

However, perhaps a better name and description could be given here. Can you consider making this change? Renaming should also be done in `default.py` and `help.py`.

```suggestion
    supports_help: ClassVar[bool] = True
    """
    Class attribute specifying whether this chat handler should show a
    help message when called with `-h/--help`.

    When set to `True` (default), `BaseChatHandler` will add a `-h/--help`
    argument on init, and implicitly handle `-h/--help` messages from users.
    """
```

Reply:
Done in 9fc238713218a728da7edd237c163e2231f0a0d0